cranelift/filetests/src/test_compile.rs

```diff
@@ -63,15 +67,25 @@ impl SubTest for TestCompile {
         info!("Generated {} bytes of code:\n{}", total_size, disasm);
 
         if self.precise_output {
-            check_precise_output(&disasm, context)
+            check_precise_output(isa, &params, &compiled_code, context)
         } else {
             run_filecheck(&disasm, context)
         }
     }
 }
 
-fn check_precise_output(text: &str, context: &Context) -> Result<()> {
-    let actual = text.lines().collect::<Vec<_>>();
+fn check_precise_output(
+    isa: &dyn isa::TargetIsa,
+    params: &FunctionParameters,
+    compiled_code: &CompiledCode,
+    context: &Context,
+) -> Result<()> {
+    let cs = isa
+        .to_capstone()
+        .map_err(|e| anyhow::format_err!("{}", e))?;
+    let buf = compiled_code.buffer.disassemble(Some(&params), &cs)?;
```

Comment:
Now that `params` is a borrow, it doesn't need to be borrowed again here. I guess Rust auto-dereferenced it for you?
```suggestion
    let buf = compiled_code.buffer.disassemble(Some(params), &cs)?;
```

Reply:
Thanks!